src/Components/CreateImageWizard/validators.ts

```diff
@@ -67,6 +67,21 @@ export const isFileSystemConfigValid = (partitions: Partition[]) => {
   return duplicates.length === 0;
 };
 
+export const isUserNameValid = (userName: string) => {
+  if (userName === undefined) return false;
```

Comment:
Should we allow the user to proceed if no data was filled in on the Users step? I mean what if I click on "Add user" because I'm curious about the functionality, but then I see that I'd need to add a password/ssh key and I change my mind. I'd be kinda stuck, right?

Reply:
The button would also work :+1: agree with whatever is the easiest right now.